core/src/main/java/de/danoeh/antennapod/core/storage/PodDBAdapter.java

```diff
@@ -558,7 +558,7 @@ public void setCompleteFeed(Feed... feeds) {
                 setFeed(feed);
                 if (feed.getItems() != null) {
                     for (FeedItem item : feed.getItems()) {
-                        setFeedItem(item, false);
+                        upsertFeedItem(item, false);
```

Comment:
Upsert?

Reply:
UPdate or inSERT

Reply:
Could you please rename to updateOrInsertFeedItem, then, please?

Reply:
I did not come up with this abbreviation. It is a common term/operation in dbms. You still want this to be expanded to full words?